autoload/neomake/makers/ft/jade.vim

```diff
@@ -0,0 +1,18 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#jade#EnabledMakers()
+    let makers = []
+    if neomake#utils#Exists('jade-lint')
```

Comment:
No need to call this yourself. Neomake won't try to invoke this maker if it doesn't exist. This function is only needed in special circumstances.